src/HTTP/ResponseBody/FileBody.php

```diff
@@ -35,7 +35,18 @@ class FileBody implements ResponseBody {
 	 * @throws InvalidFileName If the file name fails validation.
 	 */
 	public function __construct( string $filename ) {
+
```

Comment:
We should probably not be changing coding style, remove space-only changes.

Reply:
Removed empty-line change.